doc/visual_testing/visualize_reports_in_notebook.ipynb

```diff
@@ -0,0 +1,212 @@
+{
```

Comment:
I'm striggling to see where the resulting notebook can be found.

Reply:
Sorry. 

I put the notebook with the figures on a different branch. Because I don't think we want to commit that 9 MB monster to the repo. 

Also it seems that the HTML report embedded are not viewable on github so it needs to be downloaded. 

Clearly not optimal for an easy review. 

https://github.com/Remi-Gau/nilearn/blob/notebooks/doc/visual_testing/visualize_reports_in_notebook.ipynb

Reply:
It would be great to see them once ;-)